logic/src/commonMain/kotlin/com/wire/kalium/logic/sync/SyncManager.kt

```diff
@@ -30,27 +29,55 @@ interface SyncManager {
      * @see startSyncIfIdle
      * @see waitUntilSlowSyncCompletion
      */
+    @Deprecated(
+        message = "SyncManager won't serve as a Sync Utils anymore",
+        ReplaceWith(
+            expression = "syncRepository.syncState.first { it is SyncState.Live }",
```

Comment:
Seems like we will need to do this in quite few places. Would it make sense to move the `waitUntilLive` to the SyncRepository?

Reply:
Indeed.
Maybe I was too aggressive with the changes.
I've noticed this in #720 and ended up not addressing the `waitUntilLive`.

Maybe I can just remove the deprecation for this one for now? What do you think?
To be honest I don't mind this being in `SyncManager`. Maybe later we can move somewhere else if it makes sense.

Reply:
I agree, let's remove the deprecation.